### PR TITLE
Create the-solutions-journal.csl

### PR DESCRIPTION
**Pull request by jcmand**
New style definition for the solutions journal based on their guidelines: http://www.thesolutionsjournal.com/node/457

### CODE REVIEW

File: the-solutions-journal.csl [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    <link rel="self" href="http://www.zotero.org/styles/the-solutions-journal"/>
+    <link rel="template" href="http://www.zotero.org/styles/nature"/>
+    <link href="http://www.thesolutionsjournal.com/node/457" rel="documentation"/>
+    <author>
+      <name>Jafet Andersson</name>
+      <uri>http://www.smhi.se/en/research/research-departments/hydrology/jafet-andersson-1.18716</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <category field="generic-base"/>
```
Comment (adam3smith) [inline the-solutions-journal.csl]: please remove this -- we're only using this for general purpose style guides, not specific journals.

File: the-solutions-journal.csl [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      </if>
+      <else-if type="chapter" match="any"/>
+      <else>
+        <text variable="title" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author" font-variant="normal">
+      <name delimiter="," and="symbol" et-al-min="5" initialize-with="" name-as-sort-order="all"/>
```
Comment (adam3smith) [inline the-solutions-journal.csl]: please remove  the `et-al-min="5"` here. We commonly place these on the bibliography or citation, and you actually already have `et-al-min="6"`, which is correct (the instructions are "more than 5" i.e. min 6)

**Comment by adam3smith** [comment]
See the two small comments. Otherwise everything is in good shape. Instructions on updating your pull request in the first message above.

**Comment by zuphilip** [comment]
Add ISSN and EISSN
<blockquote>The ISSN for the print version of Solutions is: ISSN 2154-0896<br/>
The ISSN for the online version of Solutions is: ISSN 2154-0926</blockquote>
Source: http://www.thesolutionsjournal.com/about-us

Should the title not be just `Solutions` or maybe including the subtitle `Solutions : For a Sustainable and Desirable Future`? See also https://www.wzb.eu/de/bibliothek/bestand-recherche/elektron-zeitschriften?page=detail.phtml&bibid=WZB&colors=3&lang=en&jour_id=182678

**Comment by rmzelle** [comment]
@zuphilip, the style already has an eISSN.

@jcmand, could you change `<issnl>2154-0896</issnl>` to `<issn>2154-0896</issn>`? (we prefer print ISSN over ISSN-L for the repository)
In addition to @adam3smith requests, I also agree with @zuphilip that the style name should just be "Solutions" (without subtitle), so please change the style's file name, style ID, "self" link, and title.

**Comment by jcmand** [comment]
I have made most changes in a new commit. 
@adam3smith: I don't understand your first comment. What should I remove? Everything on line 1-16 (seems strange!) or something else?

**Comment by rmzelle** [comment]
@jcmand, just the `<category field="generic-base"/>`.

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**